pages/common/ngrep.md

```diff
@@ -0,0 +1,15 @@
+# ngrep
+
+> Ngrep is a pcap-aware tool that will allow you to specify extended regular or hexadecimal expressions to match against data payloads of packets.
+
+- Monitor all activity crossing source or destination port on network interface dev:
```

Comment:
Perhaps "source or destination" could be removed, making the sentence easier to read without losing much information?